BlueprintUI/Sources/Layout/LayoutMode.swift

```diff
@@ -4,6 +4,6 @@ public enum LayoutMode: Equatable {
     public static let `default`: Self = .strictSinglePass
 
     case standard
-    case singlePass
+    case singlePass(options: SPCacheOptions = .default)
     case strictSinglePass
```

Comment:
Feel free to discard if you don't like this pattern:

```suggestion
    case strictSinglePass
    
    public static let singlePass: Self = .singlePass()
```